testcontainers/azurite.py

```diff
@@ -0,0 +1,111 @@
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import os
+import socket
+
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.waiting_utils import wait_container_is_ready
+
+
+class AzuriteContainer(DockerContainer):
+    """
+        Azurite container.
+
+        Example
+        -------
+        ::
+
+            with AzuriteContainer() as azurite:
+               connection_string = azurite.get_connection_string()
+               BlobServiceClient.from_connection_string(
+                    connection_string,
+                    api_version="2019-12-12"
+               )
+        """
+
+    _AZURITE_ACCOUNT_NAME = os.environ.get("AZURITE_ACCOUNT_NAME", "devstoreaccount1")
+    _AZURITE_ACCOUNT_KEY = os.environ.get("AZURITE_ACCOUNT_KEY", "Eby8vdM02xNOcqFlqUwJPLlmEtlCDX"
```

Comment:
What is this account key?

Reply:
So both env variables are credentials to authenticate on the azurite container. 
"devstoreaccount1" is the default account name and "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
is the default account key. 

https://github.com/Azure/Azurite/blob/main/README.md#usage-with-azure-storage-sdks-or-tools

Reply:
Does this answer help you understand this question @tillahoffmann ?